dgraph/cmd/alpha/run_test.go

```diff
@@ -1653,13 +1653,12 @@ func TestGeoValidWkbData(t *testing.T) {
 	require.NoError(t, dg.Alter(ctx, &api.Operation{Schema: `loc: geo .`}))
 	s := `{"type": "Point", "coordinates": [1.0, 2.0]}`
 	var gt geom.T
-	if err := geojson.Unmarshal([]byte(s), &gt); err != nil {
```

Comment:
I like the existing structure better than x.Panic(). It limits the scope of `err` variable to just the if condition. It also keeps the stack trace to where the error is. It also feels more Go like.

Reply:
Let me think about this again. Gimme a day or so.

Reply:
If you want to limit the scope of `err`, you can write `x.Panic(geojson.Unmarshal([]byte(s), &gt))`